lib/PuppeteerSharp/PredefinedNetworkConditions.cs

```diff
@@ -0,0 +1,49 @@
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+
+namespace PuppeteerSharp
+{
+    /// <summary>
+    /// Predefined network conditions name.
+    /// </summary>
+    public enum PredefinedNetworkConditionsName
```

Comment:
Let's call this `NetworkCondition`

Reply:
There was an upstream debate about the singular/plural naming:
https://github.com/puppeteer/puppeteer/pull/6759#discussion_r560055871

Reply:
Good point. It's not about the plural. I don't like much the suffix `Name`what do you think?

Reply:
How about we drop the enum and declare string constants in the `NetworkConditions` class?

Reply:
Love it!

Reply:
I took inspiration from [`DeviceDescriptorName.cs`](https://github.com/hardkoded/puppeteer-sharp/blob/master/lib/PuppeteerSharp/Mobile/DeviceDescriptorName.cs)

Reply:
Those could be strings with the same values as upstream, what do you think?

Reply:
Sounds good to me

Reply:
Let's do it